client/src/services/favorites.servise.ts

```diff
@@ -0,0 +1,18 @@
+import axiosInstance from "./axios.service";
```

Comment:
`servise` typo in file name -> `service`

Reply:
done